tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
blame shows: 

https://github.com/GoogleCloudPlatform/jetty-runtime/commit/c367b6ccd98dc2c307adf3d43585f4322f1caa87

this intended usage?